system/hello_double_tap/CMakeLists.txt

```diff
@@ -2,12 +2,19 @@ add_executable(hello_double_tap
         hello_double_tap.c
         )
 
-# Double tap reset into bootrom is injected by linking with the double_tap_usb_boot library
+# Double tap reset into bootrom is injected by linking with the
+# pico_bootsel_via_double_reset library
 target_link_libraries(hello_double_tap
         pico_stdlib
-        double_tap_usb_boot
+        pico_bootsel_via_double_reset
         )
 
+# Entering the bootloader in this way also lets us specify a GPIO to be used
+# as a bootloader activity LED:
+target_compile_definitions(hello_double_tap PRIVATE
+	PICO_BOOTSEL_VIA_DOUBLE_RESET_ACTIVITY_LED=25
```

Comment:
Would there be any way to make this also use `PICO_DEFAULT_LED_PIN` rather than hard-coding `25`? Or is that a "setting" that's only available in C-land and not in cmake-land?

Reply:
It's set in the board header, so only available in C.